app/helpers/viewers_helper.rb

```diff
@@ -49,4 +49,32 @@ def get_viewers(data, data_type, obj)
     end
     []
   end
+
+  # rubocop:disable MethodLength
+  def get_viewers_input(viewers, name, translation_name, obj)
+    input = {}
+    if viewers.present?
+      checkboxes = []
```

Comment:
We should move the logic for populating checkboxes into another helper.

Reply:
How would you recommend I go about doing this?

Reply:
I would make a helper method that returns the value of checkboxes and it would contain lines 58-65.